Library/src/main/java/BookService.java

```diff
@@ -0,0 +1,109 @@
+import java.io.FileWriter;
+import java.io.IOException;
+public class BookService {
+    // to jest model
+    static boolean isWorking = true;
+    public static void doSomethingWithIt(String input) {

```

Comment:
Ta metoda powinna być w kontrolerze. No, tylko że jak ją dodasz, to ci kontroler spuchnie, może przenieś część interfejsu i pytania o rzeczy do widoku? Widok masz malutki. Kontroler ma wywoływać metody serwisowe, i obsługiwać użyszkodnika. Serwis ma się martwić operacjami na danych, a repozytorium ma dawać dostęp do danych